.github/workflows/cargo_bloat.yml

```diff
@@ -0,0 +1,83 @@
+name: Binary size report
+on:
+  push:
+    branches: [ develop ]
+    paths:
+      - 'libraries/**/*.rs'
+      - 'src/**/*.rs'
+      - 'patches/**'
+      - '**/Cargo.toml'
+      - '.cargo/config'
+      - '!third_party/**'
+  pull_request:
+    branches: [ develop ]
+    types: [opened, synchronize, reopened]
+
+jobs:
+  cargo_bloat:
+    runs-on: ubuntu-18.04
```

Comment:
Why not `ubuntu-latest`?

Reply:
I think I had an issue with some machines being more available than others, but maybe that was MacOS. Agreed latest is easier to maintain.